xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-test/xwiki-platform-attachment-test-pageobjects/src/main/java/org/xwiki/attachment/test/po/PageWithAttachmentPane.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.attachment.test.po;
+
+import org.openqa.selenium.By;
+import org.xwiki.test.ui.po.ViewPage;
+
+/**
+ * Represents a ViewPage with an attachment pane.
+ *
+ * @version $Id$
+ * @since 10.8-rc-1
```

Comment:
bad since now :) Format is also `10.8RC1`

Reply:
Added the format to https://dev.xwiki.org/xwiki/bin/view/Community/Building/#HAutomaticChecks